### ISSUE TITLE
[New Feature] Develop List TO DO

### ISSUE THREAD

**Issue by peeyee** [dolphindb/DolphinDBPlugin#20]
# To do list for new Connectors

**Comment by FlyingOE** [comment]
Item No.6 kdb+'s tasks: Is it referring to direct file access (similar to `loadFile`), bypassing q process (similar to `loadTable`)?

**Comment by peeyee** [comment]
Yes, the way of loadTable/File is totally loading the data into DolphinDB's in-memory table. Thus comes a problem if we do have enough memory，especially for community users.
So we want to have a method loadTextEx that directly loading data into DolphinDB's DFS table, meanwhile reduce the memory usage, for example, each batch costs 512MB.
